tests/js/jest.config.js

```diff
@@ -45,6 +45,14 @@ module.exports = {
 		'<rootDir>/docker/',
 		'<rootDir>/tests/e2e',
 	],
+	watchPathIgnorePatterns: [
+		'/node_modules/',
+		'/vendor/',
+		'<rootDir>/.*/build/',
+		'<rootDir>/.*/build-module/',
+		'<rootDir>/docker/',
+		'<rootDir>/tests/e2e',
+	],
```

Comment:
This is slightly unrelated to the PR but I'm always having issues when trying to run `npm run test:watch` due to the large number of files. Please let me know if I should revert this or put it in a separate PR.